### ISSUE TITLE
tar download not working 

### ISSUE THREAD

**Issue by braffes** [kjolley/BIGSdb#751]
Hi Keith,

The issue is the exactly the same as #748, but the problem is in an other file.

So this is my temporary fix.

```
index c80f77f4..0e40cf21 100644
--- a/lib/BIGSdb/SubmitPage.pm
+++ b/lib/BIGSdb/SubmitPage.pm
@@ -2480,11 +2480,7 @@ sub _tar_submission {    ## no critic (ProhibitUnusedPrivateSubroutines) #Called
        local $" = ' ';
        my $command = "cd $submission_dir && tar -cf - *";
 
-       if ( $ENV{'MOD_PERL'} ) {
-               print `$command`;    # http://modperlbook.org/html/6-4-8-Output-from-System-Calls.html
-       } else {
-               system $command || $logger->error("Can't create tar: $?");
-       }
+        print `$command` || $logger->error("Can't create tar: $?");
        return;
 }
```

Thanks.

Brice

**Comment by kjolley** [comment]
OK, thanks. Fixed as for #748.

**Comment by kjolley** [comment]
Fixed in 8d331e37c73e3aba18d05680bc79e4331c139cef
